src/libfetchers/tarball.cc

```diff
@@ -232,7 +232,7 @@ struct CurlInputScheme : InputScheme
         if (type != inputType()) return {};
 
         // FIXME: some of these only apply to TarballInputScheme.
-        std::set<std::string> allowedNames = {"type", "url", "narHash", "name", "unpack", "rev", "revCount"};
+        std::set<std::string> allowedNames = {"type", "url", "narHash", "name", "unpack", "rev", "revCount", "lastModified"};
```

Comment:
Could you list these on the protocols page, with their expected formats?

Reply:
It's already documented in src/nix/flake.md.

Reply:
It's not clear to me which attributes are allowed to be returned in the link.
Do you mean [the generic list](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-flake#flake-reference-attributes)? I don't think `dir` is allowed here though. (Maybe it should?)

The protocols page should at least refer to another page if the list of valid parameters happens to be documented there, which would be great.

Reply:
I've added lastModified to the protocol page.